### DISCUSSION TITLE
Keyboard shortcuts (and/or UI?) for fullscreen usage

### DISCUSSION THREAD

**Discussion by resle** [86Box/86Box#1923] [Ideas]
Analogue to the SDL version of Vice, the most prominent commodore 64 emulator, it would be great to have programmable keyboard shortcuts for some basic interactions with the machine such as inserting a floppy or cd, soft and hard resetting, but also shutting down entirely (with 86box itself closing).[^1]

Vice SDL goes one step further, with a petscii UI that overlays on top of full screen emulation.

The main reason for these features in my opinion is half of practical nature, and half tied to "suspension of disbelief". Not having to switch back and forth, truly has the user immersed in the emulated machine to the point of forgetting the host hardware.

[^1]: _(ps. nice feeling being able to suggest a feature without worrying about a rain of fire and brimstone ;) )_

**Comment by NixieLake** [comment]
Yes, especially for swapping media. Any time I install anything from floppy where there are several disks involved is impossible from full screen.
